core/lib/src/config/config.rs

```diff
@@ -395,6 +411,11 @@ impl Config {
             None => launch_meta_!("IP header: {}", bold("disabled"))
         }
 
+        match self.proxy_proto_header {
+            Some(ref name) => launch_meta_!("Protocol header: {}", bold(name)),
+            None => launch_meta_!("Protocol header: {}", bold("disabled"))
```

Comment:
```suggestion
            Some(ref name) => launch_meta_!("Proxy-Proto header: {}", bold(name)),
            None => launch_meta_!("Proxy-Proto header: {}", bold("disabled"))
```

Reply:
Noticed this one wasn't included yet. Did it just now